frontend/src/app/testQueue/TestQueue.test.tsx

```diff
@@ -98,41 +96,16 @@ describe("TestQueue", () => {
     expect(container).toMatchSnapshot();
   });
 
-  it("should render the new test card when feature enabled", async () => {
-    jest.spyOn(flaggedMock, "useFeature").mockReturnValue(true);
-
-    const { container } = render(
-      <MemoryRouter>
-        <MockedProvider mocks={mocks}>
-          <Provider store={store}>
-            <TestQueue activeFacilityId="a1" />
-          </Provider>
-        </MockedProvider>
-      </MemoryRouter>
-    );
-
-    await waitFor(() =>
-      expect(
-        screen.getByLabelText(
-          `Search for a ${PATIENT_TERM} to start their test`
-        )
-      )
-    );
-
-    expect(await screen.findByText("Doe, John A"));
-    expect(await screen.findByText("Smith, Jane"));
-    expect(screen.getAllByText("Submit results").length > 0).toBeTruthy();
-    expect(container).toMatchSnapshot();
-  });
-
   it("should remove items queue using the transition group", async () => {
     const { user } = renderWithUser(mocks);
     expect(await screen.findByText("Doe, John A"));
     const removeButton = await screen.findByLabelText(
       "Close test for Doe, John A"
     );
     await user.click(removeButton);
-    const confirmButton = await screen.findByText("Yes", { exact: false });
+    const confirmButton = screen.getAllByText("Yes, I'm sure", {
```

Comment:
nonblocking: curious why we're using `getAllByText` instead of `findByText`? Are there multiple cases of "Yes, I'm sure"?

Reply:
Yes, since I believe we are loading two test cards for these tests. I think something changed in how we are rendering the confirmation modal from the previous test cards so it adds all the confirmations to the DOM that are hidden from view.